esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -3313,6 +3313,31 @@ void getClusterConfig(char const * clusterType, char const * clusterName, char c
     return;
 }
 
+bool CWsWorkunitsEx::onWUGraphQuery(IEspContext &context,IEspWUGraphQueryRequest &req, IEspWUGraphQueryResponse &resp)
```

Comment:
I can add another method (maybe onWUGetGraphNames()) to get the names. Is there a way to get the names without calling the getGraphsMeta()?